package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "notifyre-nodejs-sdk",
-  "version": "1.3.0",
+  "version": "1.5.0",

```

Comment:
version can be 1.4.0
the latest release right now is 1.3.0.

Reply:
> version can be 1.4.0 the latest release right now is 1.3.0.

Thanks for letting me know, should be updated now